features/cmake/find_package/exported_targets_multiconfig/README.md

```diff
@@ -0,0 +1,61 @@
+Using unobtrusive multi-config CMake with exported targets & find_package
+=========================================================================
+
+In this example
+---------------
+
+We are using CMake to declare, export and install the targets of a couple of libraries, and using Conan to manage
+the packages and using these libraries in a project. Everything is based on the ``find_package`` feature and working with 
+multiconfig packages (Release/Debug) to avoid installing the dependencies every time we change from Debug to Release in
+the Visual Studio IDE.
+
+ - We have a "hello" library consumed by a "bye" library and a project that uses the "bye" function.
+ - The "hello" library exports its own ``helloTargets.cmake`` files, everything is pure CMake, every time we call 
+   CMake with a different build type CMake exports a ``helloTargets-{build_type}.cmake`` script.
+ - The conan recipe uses the same binary package for any ``build_type``, by removing the ``build_type`` setting.
+ - The conan recipe at the ``build`` method invokes CMake both for Release and Debug, generating for the same package.
+ - The same for the "bye" library.
+ 
+
+Some useful details of this example
+------------------------------------
+
+ - The ``CMakeLists.txt`` files for ``bye`` and ``hello`` are quite similar, the same code except the library name and
+ dependencies are changed in the first part. So you could use a similar ``CMakeLists.txt`` if you want to follow this 
+ approach.
+ - The ``helloConfig.cmake`` and ``byeConfig.cmake`` are generated by hand, almost only included the autogenerated 
```

Comment:
It is necessary to review the English, understanding this is a bit difficult.

Reply:
I rewrote some things. To make it clear that this approach is not recommended (but a possibility) and correcting some other things.